Thales/thales_safenet/ingest/parser.yml

```diff
@@ -0,0 +1,35 @@
+name: thales_safenet
+pipeline:
+  - name: deserialize
+    description: ''
+    filter: ''
+    external:
+      name: json.parse-json
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: general_value_setter
+    description: '' 
+    filter: ''
+    external: null
+stages:
+  general_value_setter:
+    actions:
+      - set:
+          event.module: 'safenet'
+          event.category: 'authentification'
+          event.code: '{{deserialize.message.details.actionText}}'
```

Comment:
`"code": "AUTH_ATTEMPT",` does not look like a code

`event.reason` or `event.action` could have been used but are used latter